src/main/resources/templates/addRecipePage.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<html lang="pl" xmlns:th="http://www.thymeleaf.org"
+      xmlns:layout="http://www.ultraq.net.nz/thymeleaf/layout"
+      layout:decorate="~{layout}">
+<main layout:fragment="content">
+<img class="rounded mx-auto d-block" width="400" alt="logo" th:src="@{/images/addRecipe.png}"> </br>
+<p><strong th:if="${newRecipe.id} ne null" th:text="|Edycja przepisu: ${newRecipe.name}|"></strong></p>
+<form method="post" th:action="@{/add-recipe}" th:object="${newRecipe}">
```

Comment:
Takie detale, ale jeżeli to jest formularz dodawania, lub edycji, to:
- action nie powinien sugerować "dodania przepisu", tylko jego zapis
- th:object nie powinien suigerować "nowego przepisu", tylko po prostu przepis